css/css-backgrounds/box-shadow-overlapping-002.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+
+  <meta charset="UTF-8">
+
+  <title>CSS Background and Borders Test: box-shadow and overlapping of adjacent text</title>
+
+  <link rel="author" title="Gérard Talbot" href="http://www.gtalbot.org/BrowserBugsSection/css21testsuite/">
+  <link rel="help" href="https://www.w3.org/TR/css-backgrounds-3/#shadow-layers" >
+  <link rel="match" href="reference/box-shadow-overlapping-001-ref.html">
+  <link rel="stylesheet" type="text/css" href="/fonts/ahem.css">
+
+  <meta content="" name="flags">
+  <meta content="Box shadows do not affect layout. Box shadows take up no space. Box shadows do not affect or influence normal flow of boxes. Therefore, box shadows can 'collide' with other boxes and can overlap other boxes (inline box or line box) or be overlapped by other boxes. In this test, the text ('PNG') follows a right outer box shadow and therefore overlaps such shadow. The box shadow, even though it is overlapped by the text ('PNG'), nevertheless overlaps the background color (yellow) of the block element." name="assert">
```

Comment:
Your test is not checking "nevertheless overlaps the background color (yellow) of the block element".

Reply:
Correct. box-shadow-overlapping-002.html does not check the "nevertheless overlaps the background color (yellow) of the block element" statement in the assert text. Another test, specifically designed to check this, could be created. Here, in box-shadow-overlapping-002.html, I will just remove this "nevertheless overlaps the background color (yellow) of the block element".